src/components/lyne-link-list/styles/lyne-link-list.component.scss

```diff
@@ -15,6 +15,14 @@ lyne-title {
   @include listReset();
 }
 
+.link-list--horizontal {
+  @include mq($from: large) {
+    display: flex;
+    flex-wrap: wrap;
+    gap: pxToRem(var(--spacing-fixed-8x));
+  }
+}
+
 ::slotted(.link-list__item:not(:last-child)) {
```

Comment:
For the horizontal variant, we would need to remove or unset the margin-block-end.

Reply:
Thanks for this one!

`margin-block-end` is unset for `.link-list--horizontal`